.pre-commit-config.yaml

```diff
@@ -57,3 +57,4 @@ repos:
         language: system
         files: \.(java|kt|gradle)$
         pass_filenames: false
+        stages: [pre-push]
```

Comment:
Why is pre-commit config changed in the same PR with a connector?

Reply:
That is a very good question, and it explains why suddenly my pre commit hook started to complain that I do not have maven install.
I tried to figure out what I did wrong but failed miserably. I've installed the pre-commit hook quite some time ago, and all is working fine.
Just before creating the pull request I had the brilliant idea of synchronizing my GitHub fork, and then rebase my local branch.
This is when the pre-commit hook started to complain about maven. In my haste to go forward with the pull request I just un-installed the pre-commit hook.
I did check master and that change is not there, so I can't explain where it came from, as I did not do that intentionally.
Anyway I'll revert the change.
Thanks for catching this!

Reply:
Restored file